wordpress-mobile/WordPress-Android
# Feature/8699 Custom vertical

Fixes #8699 

Note: This PR is targeted against `feature/8498-sc-main-vm-wizard` branch. As soon as #8675  is merged, the target branch should be changed to `feature/master-site-creation`.

Adds support for a custom category on the SiteCreation Verticals screen.

To test:
1. My Site
2. Click on Switch Site
3. Click on the plus sign button in the top right corner
4. Select Create WordPress.com site
5. Pick a segment
6. Type a query
7. Notice the last item is written in italic and contains `Custom category` subtitle
